client/themes/zenburn.css

```diff
@@ -97,12 +97,16 @@ body {
 	left: 0;
 	bottom: 0;
 	width: 220px;
+	border-radius: 0;
```

Comment:
All these theme overrides should actually be removed and design specific stuff moved from `style.css` to `example.css`. That would make for more stable themes, lighter, etc. But oh well, we already agreed on that :D
I don't want to do that until #568 makes it into master, because doing so will make the `@import` trick in custom CSS not valid anymore, so users couldn't reliably change their themes at all.